### PR TITLE
RIP-826 Updating index.js to reflect changes to explorance integration

### PR DESCRIPTION
**Pull request by mlincoln**
https://jira-secure.berkeley.edu/browse/RIP-826

### CODE REVIEW

File: public/canvas-customization/index.js [hunk 1]
```diff
@@ -36,16 +36,16 @@
...
+    canvasAPI:window.location.origin,
+    domainName:"com.explorance",
+    consumerID:"CWN1RxHsa4vY9b2QgVC7jQ==",
+    defaultLanguage:"en-us"
+},
     BlueCanvasJS=document.createElement("script");
 BlueCanvasJS.setAttribute("type","text/javascript");
 BlueCanvasJS.setAttribute("src","https://surveys.berkeley.edu/berkeleySurveysblueconnector//Scripts/Canvas/BlueCanvas.min.js");
-document.getElementsByTagName("head")[0].appendChild(BlueCanvasJS);
+BlueCanvasJS.async=!0;document.getElementsByTagName("head")[0].appendChild(BlueCanvasJS);
```
Comment (johncrossman) [inline public/canvas-customization/index.js]: @mlincoln - Would a newline after the semi-colon make this more readable?

Comment (johncrossman) [reply public/canvas-customization/index.js]: @mlincoln - Also, the expression `!0` will always be true. Is this syntax recommended?

Comment (mlincoln) [reply public/canvas-customization/index.js]: After line 47?

Comment (johncrossman) [reply public/canvas-customization/index.js]: I'm suggesting a line break after `BlueCanvasJS.async=!0;`

Comment (mlincoln) [reply public/canvas-customization/index.js]: Thanks, I added a line break after that semicolon, but I left the !0 piece as-is--I'm just copy/pasting code from Explorance.

**Comment by johncrossman** [comment]
Thanks @mlincoln

File: public/canvas-customization/index.js (hunk 1 above)
Comment (pauline2k) [reply public/canvas-customization/index.js]: As a consequence of this PR I learned that `!0` and `!1` are standard Javascript minifications for expressing `true` and `false` in the fewest possible characters. In general we don't bother with minifying our custom Canvas JS, but having it in there won't hurt anything, other than perhaps confusing future generations.

**Pull request merged**